ksqldb-rest-app/src/test/java/io/confluent/ksql/rest/integration/KsqlResourceFunctionalTest.java

```diff
@@ -62,9 +62,12 @@
 import org.junit.Test;
 import org.junit.experimental.categories.Category;
 import org.junit.rules.RuleChain;
+import org.junit.runner.RunWith;
+import org.mockito.junit.MockitoJUnitRunner;
 
 @SuppressWarnings("unchecked")
 @Category({IntegrationTest.class})
+@RunWith(MockitoJUnitRunner.class)
```

Comment:
Just for my own information, what does this change?

Reply:
If this class calls out to a mockito.Mock somehow, it won't cause havoc in intellij